demo/src/constants/buttons.ts

```diff
@@ -118,7 +118,7 @@ export const buttonLabelMap = {
   'shape-rectangle': 'Shape Rectangle',
   'shape-line': 'Shape Line',
   text: 'Add Text Tool',
-  'enhanced-stereo': 'Stereochemistry',
+  'enhanced-stereo': 'Stereochemistry'
 }
```

Comment:
I think everything's working great! Maybe let's add an option to hide this button into our demo app right away?

```diff
...
  text: 'Add Text Tool',
  'enhanced-stereo': 'Stereochemistry',
+ fullscreen: 'Full Screen'
}
```

Also need to add 'fullscreen' string into `allButtons` array above.

Reply:
added

Reply:
added